client/js/widgets/button.js

```diff
@@ -24,7 +24,8 @@ export class Button extends Widget {
       borderColorOH: null,
       textColorOH:null,
 
-      text: ''
+      text: '',
+      borderRadius: 800
```

Comment:
It's just a default value. Setting a large fixed value is an easy way to make a pill shape.

Reply:
yeah but I was saying that we could probably change that to %

Reply:
No, if it is a percentage, then the buttons will be ellipses instead of pills. Percentage is calculated based on width for x radius and height for y radius.

Reply:
oh yeah!! please ignore my stupidity :)